modules/swift/examples/code_snippets/SampleCodeTest.swift

```diff
@@ -120,12 +108,25 @@ class SampleCodeTest {
         Database.log.file.level = .info // <.>
         // end::file-logging[]
     }
+    
+    func dontTestNewFileLogging() throws {
+        // tag::new-file-logging[]
+        let tempFolder = NSTemporaryDirectory().appending("cbllog")
+        LogSinks.file = FileLogSink(level: .info, directory: tempFolder)
```

Comment:
Just noticed this from Java Snippet. It is a good idea to be consistent.

```
LogSinks.get().setFile(new FileLogSink.Builder()
            .setDirectory("/tmp/logs")
            .setMaxKeptFiles(12)
            .setPlainText(false)
            .build());
```

Reply:
I also think it's a good idea to set the log level to verbose as that is what we want users to do.